docs/content/preview/explore/going-beyond-sql/connection-mgr-ysql.md

```diff
@@ -55,7 +55,7 @@ When `enable_ysql_conn_mgr` is set, each YB-TServer starts the YSQL Connection M
 To create a single-node cluster with YSQL Connection Manager using [yugabyted](../../../reference/configuration/yugabyted/), use the following  command:
 
 ```sh
-./bin/yugabyted start --tserver_flags "enable_ysql_conn_mgr=true,allowed_preview_flags_csv=enable_ysql_conn_mgr" --ui false
+./bin/yugabyted start --tserver_flags "enable_ysql_conn_mgr=true,allowed_preview_flags_csv={enable_ysql_conn_mgr}" --ui false
```

Comment:
LGTM, but this uses double quotes, thread example uses single quotes, does it make a difference

Reply:
AFAIK both should work in linux.